graphql_api/types/0_TEMP/coverage.graphql

```diff
@@ -0,0 +1,51 @@
+# Coverage is information related to a repo's
+# main branch test coverage
+type CoverageAnalysis {
+  hits: Int
+  misses: Int
+  lines: Int
+  coverageSha: String
+  percentCovered: Float # formerly repo.coverage
+
+  #  # TODO - not complete alignment with this type (missing partialsCount, fileCount) - what to do
+  #  # formerly repo.coverage, repo.hits, repo.misses
+  #  coverageTotals: CoverageTotals
+
+  # formerly repo.measurements
+  coverageMeasurements(
+    interval: MeasurementInterval!
+    after: DateTime
+    before: DateTime
+    branch: String
+  ): [Measurement!]!
+
+  ### FLAGS ###
```

Comment:
Do we want Flags and Components to have a subtype, something like below? To futureproof it getting added to TestAnalysis or BundleAnalysis as filters in the future
```
type FlagMeasurements {
   flagsCount: Int!
...
}
```

Reply:
The schema under "Another option is" looks so clean! But yeah right now there's some small nuances between different types of measurements (eg coverage, flag, component, bundle analysis) that will require alot of refactoring to make it happen :(